docs/exampleLink.mdx

```diff
@@ -0,0 +1,7 @@
+export const ExampleLink = ({component}) =>(
+	<>
+		<h2> Beispiele ansehen </h2>
+  	<a href ={`https://public-ui.github.io/v2/sample-react/#/${component}`}>Hier klicken, um das Biespiel der {component}-Komponente anzusehen </a>
```

Comment:
Anstatt den Link auf `v2` zu hardcoden, sollten wir die ausgewählte Version der Docs verwenden.

Beispiel: https://github.com/public-ui/public-ui.github.io/blob/main/src/components/docs/ComponentList.tsx#L74

@deleonio Legst du Wert darauf, dass die Beispiel-Links auch in älteren Versionen nachgepflegt werden? Gerade ist es so eingerichtet, dass die Links ab der "nächsten Version" immer vorhanden wären. Das fände ich persönlich ausreichend.

Reply:
Nein. Die aktuelle Hauptversion 2.1 sollte möglichst perfekt werden.

Reply:
@anicyne Das heißt, wir müssen einmal die mdx-Dateien in den 2.1 Docs analog zu den aktuellen Docs anpassen.

Ordner: `versioned_docs/version-2.1/30-components`
Hier bitte jeweils die ExampleLink Komponenten einfügen
Für den Import dann am besten das `@site`-Alias verwenden. In `docs` _und_ `versioned_docs`: `@site/src/components/LExampleLink`